### DISCUSSION TITLE
Support international domain names

### DISCUSSION THREAD

**Discussion by tigransimonyan** [mastodon/mastodon#16682] [Q&A] [answered]
Hello guys,

I created an instance of a mastodon using an international domain name and realized that it was not working correctly. For example my friends from another instance [can not find me](https://github.com/mastodon/mastodon/issues/16691), [mention or write a direct message](https://github.com/mastodon/mastodon/issues/16602), etc. 

Of course, I could register a domain with Latin letters, but I think IDN is also about decentralization. 

Do you have a instance with IDN? What do you think about this?

**Comment by ClearlyClaire** [comment, marked as answer]
Those were bugs with IDN TLDs (IDN domain names with ascii TLDs should work as expected). This will be fixed in the next release.

**Comment by tigransimonyan** [comment]
When are you planning to release the new version?

I want to fix this problem for myself and my friends, but I'm not sure if I can do it without instructions :/
